cmd/lib.go

```diff
@@ -17,17 +17,11 @@ var Exit func(int) = func(code int) { os.Exit(code) }
 var ssh string
 var defaultSsh = ""
 
-var dockerContainer string
-var defaultDockerContainer = ""
+var docker string
```

Comment:
nit: do you mind changing all of these to match the long format option it refers to, for consistency?

```suggestion
var targetDocker string
```

Reply:
It this is changed, makes sense to keep consistency across variables and call 
-`targetSSH`
-`targetDocker`
-`targetLocalhost`

Reply:
remember, nit == take it or leave it, no problem.